Sources/RxKeyboard.swift

```diff
@@ -29,7 +29,10 @@ public class RxKeyboard: NSObject {
   /// Same with `visibleHeight` but only emits values when keyboard is about to show. This is
   /// useful when adjusting scroll view content offset.
   public let willShowVisibleHeight: Driver<CGFloat>
-
+  
+  /// An observable visibility of keyboard. Emits keyboard visibility
+  /// when changed keyboard show and hide.
+  public let didChangeVisibility: Driver<Bool>
```

Comment:
Looks cool. How about using `isHidden` or `isVisible` instead? I think the meaning of 'didChange' is already included in Observable itself.